airflow/executors/celery_executor.py

```diff
@@ -125,8 +129,10 @@ def _execute_in_fork(command_to_exec: CommandType) -> None:
         os._exit(ret)
 
 
-def _execute_in_subprocess(command_to_exec: CommandType) -> None:
+def _execute_in_subprocess(command_to_exec: CommandType, celery_task_id: Optional[str] = None) -> None:
     env = os.environ.copy()
+    if celery_task_id:
+        env["celery_task_id"] = celery_task_id
```

Comment:
@ephraimbuddy this is the missing environment variable being set :)

Reply:
Looks like we should do `env['external_executor_id'] = celery_task` here, right 🤔?

Reply:
You are absolutely right 🥇 
I will add tests ones I find the time this week.

Reply:
This should now be fixed :)